Doc/library/argparse.rst

```diff
@@ -26,6 +26,61 @@ module also automatically generates help and usage messages and issues errors
 when users give the program invalid arguments.
 
 
+Summary
+-------
+
+Core function
```

Comment:
s/function/functions/ ?

Also ``ArgumentParser`` is a class, not a function.

Reply:
Maybe «Core functionality» was meant.